Lib9c/Action/WeeklyArena.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace Nekoyume.Action
 {
+    // NOTE: This class not used anymore.
```

Comment:
```suggestion
    // FIXME: This class isn't used anymore.
```

or... It might be better to delete it now.